### PR TITLE
fix(ui-ux): contact tab bug fixes

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What this PR does / why we need it:
Under the contracts tab, using `MarbleLsdV1` and `MarbleLsdProxy` contract:

1. Fix boolean value display under Read Contract tab
image

2. Try deposit (View Transaction redirects to mainnet) 
4. Fix setWithdraw, setDeposit boolean on Write As Proxy 
5. Fix claimWithdrawals to accept input of array 
image

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
Most of the write methods can only tested with the contract creator's address

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

**Comment by fullstackninja864** [comment]
Steps:
1. visit [preview](https://deploy-preview-385--defimetascan.netlify.app/address/0x0B52a71A03a47246BD9d9C556B6Df9C42e73462A?network=TestNet)
2. go to contracts > Read as proxy > click on expand all
image

**Comment by fullstackninja864** [comment]
cannot add array values in write proxy
image

**Comment by chloezxyy** [comment]
> 1. preview

Just to clarify, this is in Testnet network and also under MarbleLsdProxy Contract? This is what im seeing

image

**Comment by chloezxyy** [comment]
> cannot add array values in write proxy image

Is this the expected error?
image

**Comment by fullstackninja864** [comment]
> > 1. preview
> 
> Just to clarify, this is in Testnet network and also under MarbleLsdProxy Contract? This is what im seeing
> 
> image

yes now click on expand all

**Comment by fullstackninja864** [comment]
> > cannot add array values in write proxy image
> 
> Is this the expected error? image

[2] is valid value, check below link to compare
https://sepolia.etherscan.io/address/0x49A3e882B8Ca7BA93CD412D8dbF8b5f0C9f8FD94#writeProxyContract

**Comment by chloezxyy** [comment]
> cannot add array values in write proxy image

This seems like a separate/new issue - looking into it

**Comment by chloezxyy** [comment]
> > > cannot add array values in write proxy image
> > 
> > 
> > Is this the expected error? image
> 
> [2] is valid value, check below link to compare https://sepolia.etherscan.io/address/0x49A3e882B8Ca7BA93CD412D8dbF8b5f0C9f8FD94#writeProxyContract

Hmm but this is using sepolia network? But metachain is using our defichain testnet network - will check on this

**Comment by fullstackninja864** [comment]
its not related to chain as using remix its working fine 
image

**Comment by chloezxyy** [comment]
> its not related to chain as using remix its working fine image

You're right, its an error caused by how the input arguments were being processed, added into list of fixed issues

File: apps/web/src/pages/address/_components/contract/ContractMethodForm.tsx [hunk 1]
```diff
@@ -62,15 +64,40 @@ export default function ContractMethodForm({
...
+        // parse the string into an array
+        return JSON.parse(value);
+      }
+
+      if (inputType === "bool") {
+        return value === "true";
+      }
+      if (inputType === "uint256") {
+        return parseEther(value);
+      }
```
Comment (fullstackninja864) [inline apps/web/src/pages/address/_components/contract/ContractMethodForm.tsx]: Lets remove this,

Comment (chloezxyy) [reply apps/web/src/pages/address/_components/contract/ContractMethodForm.tsx]: removed

**Review by fullstackninja864 — APPROVED** [review]

**Pull request merged**